mulighetsrommet-tiltakshistorikk/src/main/resources/db/migration/V4__gruppetiltak.sql

```diff
@@ -0,0 +1,34 @@
+create type gruppetiltak_tiltakskode as enum (
+    'AVKLARING',
+    'OPPFOLGING',
+    'GRUPPE_ARBEIDSMARKEDSOPPLAERING',
+    'JOBBKLUBB',
+    'DIGITALT_OPPFOLGINGSTILTAK',
+    'ARBEIDSFORBEREDENDE_TRENING',
+    'GRUPPE_FAG_OG_YRKESOPPLAERING',
+    'ARBEIDSRETTET_REHABILITERING',
+    'VARIG_TILRETTELAGT_ARBEID_SKJERMET');
+
+create type gruppetiltak_status as enum (
+    'PLANLAGT',
+    'GJENNOMFORES',
+    'AVSLUTTET',
+    'AVBRUTT',
+    'AVLYST');
+
+create type gruppetiltak_oppstartstype as enum (
+    'LOPENDE',
+    'FELLES');
+
+
+create table gruppetiltak
```

Comment:
Det er kanskje bedre å kalle dette for `gjennomforing`

Reply:
Likte egentlig gruppetiltak, men siden alle andre steder hos oss heter det gjennomføring... Når vi tar over enkamo f. eks angrer man på gruppetiltak navnet da?

Reply:
Jeg liker også gruppetiltak